### ISSUE TITLE
Identifcation of experimental features/items?

### ISSUE THREAD

**Issue by Nuri0** [PrismarineJS/minecraft-data#847]
The version 1.20.3 contained some items that are not part of the main game, but are only available with the "Update 1.21" switch / data-pack (e.g. "crafter" or "trial_key").

https://github.com/PrismarineJS/minecraft-data/blob/master/data/pc/1.20.3/items.json

These items are listed as if they were part of the main game or is there some indication, which marks these items as "experimental" or "preview" that I am missing?

**Comment by extremeheat** [comment]
No, there's no classification for that currently, minecraft-data includes every block and item a client/server supports for any specific version, as it's obtained with a generator script running on Minecraft. But PRs are open if you want to add them, what is your use case?

**Comment by Nuri0** [comment]
I wanted to generate kind of a history of minecraft items. For this I planned to group items by the version they were added in - simply by looking at diff of consecutive versions. Technically this still works for the mentioned items/blocks as they were added in 1.20.3 (23w42a) even if they are "hidden" behind the flag.

**Comment by rom1504** [comment]
I think @wgaylord was looking into this recently

**Comment by wgaylord** [comment]
I will say they WERE added as they always take up space in the block registry even with the flag off, you just can't access them with it off. (Which is why they are included.)  Annoyingly there is no good way to automatically grab what feature blocks or items are part of :(
